src/main/resources/META-INF/plugin.xml

```diff
@@ -78,7 +78,7 @@
                 description="Refresh Liberty Dev Dashboard projects" icon="AllIcons.Actions.Refresh"/>
 
         <action id="io.openliberty.tools.intellij.actions.RunLibertyDevTask"
-                class="io.openliberty.tools.intellij.actions.RunLibertyDevTask" text="Run Liberty Dev Task"
+                class="io.openliberty.tools.intellij.actions.RunLibertyDevTask" text="Run Liberty Dev Action"
                 description="Execute the selected liberty dev task" icon="AllIcons.Actions.Execute"/>
```

Comment:
Perhaps "Run Liberty action", don't need to mention dev.  Also update the description to match e.g. "Run the selected Liberty action".